src/directory/directory.mjs

```diff
@@ -1534,6 +1529,61 @@ export const directory = {
           }
         ]
       },
+      'push-notifications': {
```

Comment:
Unlike other categories, push notifications only existed for "v1" (v5) of the React Native platform - for all others platforms, PN was only introduced as part of "current"